### ISSUE TITLE
When downloading using "embeddedsubtitles" with "HI fallback", HI subtitles are not downloaded

### ISSUE THREAD

**Issue by Franklin22221** [morpheus65535/bazarr#2195]
When downloading with embeddedsubtitles provider while "HI fallback" is on, HI\sdh subs are not downloaded and are crossed out with yellow X when searching manually - 
![image](https://github.com/morpheus65535/bazarr/assets/138801412/6f11d53d-4d9a-4ee6-82ad-eaa579dcd837)

**To Reproduce**
1. Make sure that Settings>subtitles>Use embedded subs is OFF
2. go to Settings>providers
3. add embeddedsubtitles provider
4. click on it, Make "Use HI as fallback" TRUE
5. auto-Search for subs

**Expected behavior**
When auto searching episodes that have only english HI subs - fallback to it and download it.

[bazarr (1).log](https://github.com/morpheus65535/bazarr/files/11970013/bazarr.1.log)

Bazarr Version 1.2.3-beta.2
Sonarr Version 3.0.10.1567
Radarr Version 4.6.4.7568 
Operating System Windows-10

**Additional context**
1. These episodes have only 1 english result that is HI and are ignored. Manual download of these subs work well.
2. The selected profile has one language - english, with Normal or HI selected.
3. Episodes with both reg and HI english subs are working well (for reg)
4. Log included.

Hopefully thats everything, Thanks!

**Comment by morpheus65535** [comment]
@vitiko98 could you take a look please?

**Comment by EmilianoNeme** [comment]
I suffer from the same bug. Using "Language = English" and "Subtitles Type = Normal or hearing impaired" as my unique language profile and "Embedded Subtitles" as only Provider, automatic search wouldn't work, but manual would show subs with 99% rating, being the HI property the one preventing them from being a perfect 100% match.
**Something that works for me is the "Language Equals" option.  Setting "English [HI]" as the "Source" and "English" as the "Target".** 
But maybe that isn't suitable for other people setups. I, In my case, have multiple Bazarr instances (*) and I run 2 of them with the "Hearing impaired" Post-Processing option so my media don't show HI tags.

I also tried creating another Language Profile. I used "Language = English" and "Subtitles Type = Hearing impaired required". Assigning that profile to the series for which Bazarr couldn't provide automatic subtitles, Bazarr is now able to. However because of the  "Hearing impaired" Post-Processing they end up as "En" subs instead of "En Hi", so Bazarr still shows them as wanted, even though they are not. Once subtitles are downloaded, changing the Language Profile back to the original one with "Subtitles Type = Normal or hearing impaired" resolves it. However this is not as nice as using "Language Equals".

I even change the "series_scores" in the config.ini file, replacing "hash = 359 " for "hash= 360" and "hearing_impaired = 1" for "hearing_impaired = 0", this makes Bazarr shows the embedded subs HI subs with a 100% score, but they still won't download automatically. 

*Since it is not possible to assign different providers and post-processing options to different language profiles I run more than once instance of Bazarr and I have "Languages -> Default Settings" disabled on all of them.
I have a specific instance for Web-Dl movies / films that already include high quality subs, but many times only HI. So I manually add a profile I named "En-Web" to those, and leave others without any profile, instead I assign them a profile in their respective instance.
My other instance is for DVD/Blu-ray movies and shows, with multiple providers, and I use the "Download Monitored Only" option for movies, and I set special edition films, like extended cuts, as unmonitored, so I can download their subtitles manually, since there are cases when there are not subs for rare movies or the are not right matches (Bazarr might download a normal subtitle for an extended edition is the latter isn't available, which is wrong).
